src/python/pants/backend/go/target_types.py

```diff
@@ -22,9 +22,51 @@
     MultipleSourcesField,
     StringField,
     Target,
+    TargetGenerator,
     ValidNumbers,
 )
 
+# -----------------------------------------------------------------------------------------------
+# `go_third_party_package` target
```

Comment:
The sections were reordered in order for `GoThirdPartyPackageTarget` to be declared before `GoModTarget`.